kebechet/managers/manager.py

```diff
@@ -46,6 +50,24 @@ def _init_service_url(service_type: ServiceType = None, service_url: str = None)
     return service_url
 
 
+def refresh_repo_url(decorated: Any):  # noqa: N805
+    """Check if access token has expired and refresh repo url if necessary."""  # noqa: D202
+    # noqa: D202
```

Comment:
Does it fight with black on formatting?

Reply:
I tried updating pydocstyle, seems like black wants a new line, where as pydocstyle doesnt like the new line.